package.json

```diff
@@ -17,7 +17,8 @@
     "build:style-dictionary": "node style-dictionary.js && npm run copy:scss",
     "build:patternlab-css": "sass source/css/patternlab.scss:out/css/patternlab.css --style=compressed --embed-sources --load-path=node_modules",
     "build:tailwind": "node scripts/tailwind-config-generator.mjs false true && cpr scripts/tailwind-config-generator.mjs build/tailwind/tailwind-config-generator.mjs -o",
-    "build-test:css": "sass build/scss/_db-ui-base.scss:build/css/db-ui-base.css --load-path=node_modules",
+    "build-test:base-css": "sass build/scss/_db-ui-base.scss:build/css/db-ui-base.css --load-path=node_modules",
+    "build-test:color-css": "sass build/scss/_color-classes.scss:build/css/color-classes.css --load-path=node_modules",
```

Comment:
You could rename those SCSS partial files by removing the underscore in the beginning and then still use the original script with a slight modification to transform all non-partial files within this folder:
```
"build-test:css": "sass build/scss/:build/css/ --load-path=node_modules",
```

compare to e.g. https://github.com/db-ui/core/blob/main/package.json#L35

Reply:
fixed